schemas/oic.r.account-request.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema" : "http://json-schema.org/draft-04/schema#",
+  "description" : "Copyright (c) 2017 Open Connectivity Foundation, Inc. All rights reserved.",
+  "title" : "Account",
+  "id": "https://www.openconnectivity.org/ocf-apis/security/schemas/oic.r.account-request.json#",
+  "definitions" : {
+    "uuid" : {
+      "type" : "string",
+      "pattern" : "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
```

Comment:
We should reference the definition of a UUID in the oic.types-schema.json rather than also defining it here.  Maybe Mark can help with the appropriate way to reference a schema in a different Git repo if you're not sure how to do it.

Reply:
Done.